scan

```diff
@@ -80,13 +80,15 @@ def scan_domains(scanners, domains):
             'writer': scanner_writer
         }
 
-    # For each domain, run through each scanner.
```

Comment:
Flipping the loop structure is a major change that I'd prefer to tackle separately. 

Additionally, this project is moving towards parallelizing the scanning process (#16), so keeping open a single instance of phantomjs is not desirable. I'd rather accept an annoyingly slow Google Analytics scanner, and use that as an impetus to complete the parallelization process, which will make the whole process a *lot* faster for both this new scanner and the existing scanners.